pkgs/top-level/all-packages.nix

```diff
@@ -17284,6 +17284,8 @@ with pkgs;
 
   smlpkg = callPackage ../tools/package-management/smlpkg { };
 
+  smlfut = callPackage ../development/tools/smlfut { };
```

Comment:
Consider moving to `pkgs/by-name/sm/smlpkg/package.nix` instead.

Reply:
Done.  I take it this is new best practice?  Fine by me; the old categories often seemed contrived anyway.